pokedex/pokemon/views.py

```diff
@@ -99,6 +100,57 @@ def delete_pokemon_info(request: HttpRequest, national_num: int) -> HttpResponse
     pokemon_info.delete()
     return HttpResponse(json.dumps({}))
 
+# ===POKEMON_TYPE===
+def find_all_pokemon_types(_: HttpRequest) -> HttpResponse:
+    pokemon_types = PokemonType.objects.all()
+    return HttpResponse(to_json(pokemon_types))
+    
+def find_all_pokemon_types_by_pokemon_id(_: HttpRequest, national_num: int) -> HttpResponse:
+    pokemon_types = PokemonType.objects.all(pokemon_info__pk=national_num)
+    return HttpResponse(to_json(pokemon_types))
+
+def find_pokemon_type_by_pokemon_id_and_type(_: HttpRequest, national_num: int, type: str) -> HttpResponse:
+    pokemon_type = get_object_or_404(PokemonType, type=type, pokemon_info__pk=national_num)
+    return HttpResponse(to_json_one(pokemon_type))
+    
+# The body request should include a national_num identifying which pokemoin_info we are looking at 
```

Comment:
nit: pokemon_info

Reply:
nit?

Reply:
nitpick- small thing that doesn't affect the PR quality but is just a quick fix

Reply:
ooo what do u want me to change it to?

Reply:
it's a typo haha: `pokemoin_info` to `pokemon_info`

Reply:
OMGGGG 
okok let me change